packages/multicont/multicont.1.0.0-rc.1/opam

```diff
@@ -7,7 +7,7 @@ homepage: "https://github.com/dhil/ocaml-multicont"
 dev-repo: "git+https://github.com/dhil/ocaml-multicont.git"
 bug-reports: "https://github.com/dhil/ocaml-multicont"
 license: "MIT"
-version: "1.0.0-rc.1"
+version: "1.0.0~rc.1"
```

Comment:
```suggestion
```
The version field is not used in opam-repository. The version number is already present in the directory structure. You need to `git mv` the directory instead and remove this field

Reply:
I use the version available during the install atm. Although, I could potentially make a script to generate the .install as suggested above.

Reply:
The opam file is not copied in the build directory IIRC so removing it wouldn’t do anything to your package